packages/core/src/drizzle/index.ts

```diff
@@ -225,6 +219,8 @@ export const onchainTable = <
   extra: extra;
   dialect: "pg";
 }> => {
+  // @ts-ignore
+  const instanceId: string | undefined = globalThis.__PONDER_INSTANCE_ID;
```

Comment:
Could add a `declare global` somewhere in `packages/core` to annotate this as `string | undefined`, would remove the need for `@ts-ignore` everywhere

https://stackoverflow.com/a/69429093

Reply:
Hmm this wasn't working for me